content/fr/community/_index.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Community"
+title: "Communauté"
 ---
 
 <section class="section">
   <div class="container">
     <div class="row align-items-center">
       <div class="col-md-6 order-2 order-md-1">
-        <p>Welcome to the Community page - we are so happy you are here! All our learning resources and events are open to everyone and are free to use. However, the magic happens when the community comes together to share learnings. We hope you will join us!
+        <p>Bienvenue sur la page de la Communauté - nous sommes heureux que vous soyez ici ! Toutes nos ressources d'apprentissage et tous nos événements sont ouverts à tous et leur utilisation est gratuite. Toutefois, la magie opère lorsque la communauté se réunit pour partager ses connaissances. Nous espérons que vous vous joindrez à nous !
```

Comment:
suggestion: nous sommes heureux de vous voir ici

Reply:
Taken into account in next commit